README.md

```diff
@@ -1,536 +1,121 @@
-* `Crash-<sha1_of_input>.java` contains a class with a `main` function that invokes the fuzz target with the
-  crashing input. This is especially useful if using `FuzzedDataProvider` as the raw bytes of the input do not
-  directly correspond to the values consumed by the fuzz target. The `.java` file can be compiled with just
-  the fuzz target and its dependencies in the classpath (plus `jazzer_api_deploy.jar` if using `FuzzedDataProvider).
+## Using Jazzer via...
 
-### Minimizing a crashing input
+### JUnit 5
 
-Every crash stack trace is accompanied by a `DEDUP_TOKEN` that uniquely identifies the relevant parts of the stack
-trace. This value is used by libFuzzer while minimizing a crashing input to ensure that the smaller inputs reproduce
-the "same" bug. To minimize a crashing input, execute Jazzer with the following arguments in addition to `--cp` and
-`--target_class`:
+The following steps assume that JUnit 5 is set up for your project, for example based on the official [junit5-samples](https://github.com/junit-team/junit5-samples).
 
-```bash
--minimize_crash=1 <path/to/crashing_input>
-```
-
-### Parallel execution
-
-libFuzzer offers the `-fork=N` and `-jobs=N` flags for parallel fuzzing, both of which are also supported by Jazzer.
-
-### Limitations
+1. Add a dependency on `com.code-intelligence:jazzer-junit:<latest version>`.
+2. Add a new *fuzz test* to a new or existing test class: a method annotated with [`@FuzzTest`](https://codeintelligencetesting.github.io/jazzer-docs/jazzer-junit/com/code_intelligence/jazzer/junit/FuzzTest.html) and at least one parameter.
+   Using a single parameter of type [`FuzzedDataProvider`](https://codeintelligencetesting.github.io/jazzer-docs/jazzer-api/com/code_intelligence/jazzer/api/FuzzedDataProvider.html), which provides utility functions to produce commonly used Java values, or `byte[]` is recommended for optimal performance and reproducibility of findings.
+3. Assuming your test class is called `com.example.MyFuzzTests`, create the *inputs directory* `src/test/resources/com/example/MyFuzzTestsInputs`.
+4. Run a fuzz test with the environment variable `JAZZER_FUZZ` set to `1` to let the fuzzer rapidly try new sets of arguments.
+   If the fuzzer finds arguments that make your fuzz test fail or even trigger a security issue, it will store them in the inputs directory.
+5. Run the fuzz test without `JAZZER_FUZZ` set to execute it only on the inputs in the inputs directory.
+   This mode, which behaves just like a traditional unit test, ensures that issues previously found by the fuzzer remain fixed and can also be used to debug the fuzz test on individual inputs.
 
-Jazzer currently maintains coverage information in a global variable that is shared among threads. This means that while
-fuzzing multi-threaded fuzz targets is theoretically possible, the reported coverage information may be misleading.
+A complete Maven example project can be found in [`examples/junit`](examples/junit).
```

Comment:
Add a code snippet of a fuzz test to show how similar it looks to normal JUnit tests?